src/main/java/uk/gov/ons/census/casesvc/utility/FieldworkHelper.java

```diff
@@ -1,21 +1,66 @@
 package uk.gov.ons.census.casesvc.utility;
 
+import org.springframework.util.StringUtils;
 import uk.gov.ons.census.casesvc.model.entity.Case;
 
 public class FieldworkHelper {
-  public static boolean shouldSendCaseToField(Case caze, String eventChannel) {
-    if (eventChannel.equals("FIELD")) {
+  public static boolean shouldSendCaseToField(Case caze) {
+
+    if (StringUtils.isEmpty(caze.getEstabType())
+        || caze.getEstabType().equals("TRANSIENT PERSONS")) {
       return false;
     }
+
     if (caze.getRegion().startsWith("N") && caze.getCaseType().equals("CE")) {
       return false;
     }
-    if (caze.getEstabType().equals("TRANSIENT PERSONS")) {
+
+    if (caze.isAddressInvalid()) {
       return false;
     }
+
     if (caze.getRefusalReceived() != null) {
       return false;
     }
+
+    if (caze.isReceiptReceived()) {
+      if (!caze.getCaseType().equals("CE")) {
+        return false;
+      }
+
+      if (!caze.getAddressLevel().equals("E")) {
+        return false;
+      }
+    }
+
+    if (caze.isReceiptReceived()
+        && caze.getCaseType().equals("CE")
+        && caze.getAddressLevel().equals("E")
+        && (caze.getCeActualResponses() >= caze.getCeExpectedCapacity())) {
```

Comment:
this is not a safe operation. `ceExpectedCapacity` is nullable so this will cause null pointer exceptions

Reply:
I thought that for a CE E it should always be set unless we've got a bigger data integrity issue?

Reply:
ugh. okay leave it then. I'm not a fan of the proliferation of places in the code which have the potential to NPE

Reply:
changed it